crates/bevy_color/src/oklaba.rs

```diff
@@ -216,7 +216,10 @@ impl ColorToComponents for Oklaba {
     }
 }
 
-#[allow(clippy::excessive_precision)]
+#[expect(
+    clippy::excessive_precision,
+    reason = "The code with excessive precision was copied from another codebase."
+)]
```

Comment:
I'm not really happy with this lint attribute (as well as the other one changed in this file), as there really isn't a reason for the floats to have excessive precision, beyond the code having been copied verbatim from another code-base.

That said, I feel truncating the floats is out-of-scope for this PR, hence me changing them to be `expect(...)`.